src/dotnet/commands/dotnet-test/DotnetTestParams.cs

```diff
@@ -57,6 +59,8 @@ public DotnetTestParams()
             };
 
             AddDotnetTestParameters();
+
+            Help = true;
```

Comment:
I get why this works, but it's not obvious. We need to fix this in System.CommandLine